docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -156,6 +156,10 @@ DKIM is currently supported by either OpenDKIM or Rspamd:
         use_esld = true;
         check_pubkey = true; # you want to use this in the beginning
 
+        selector = "mail";
+        path = "/tmp/docker-mailserver/opendkim/keys/$domain/$selector.private"; # this will automatically match keys for domains
```

Comment:
- I've moved the comment above the `path` setting and revised it for context.
- The `path` was pointing to `opendkim`, it was probably meant to point to `rspamd`? I've normalized it to `dkim` which we'd like to switch to (_perhaps for v14_) Feel free to set it to `rspamd` instead for now, which would be consistent with the `path` config that follows afterwards.

```suggestion
        selector = "mail";
        # The location is searched for a DKIM key with `$domain` sourced from the mail `FROM` header, while `$selector` is configured for `mail` as a default.
        path = "/tmp/docker-mailserver/dkim/keys/$domain/$selector.private";
```